low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -269,7 +278,9 @@ def process(self):
             # from cortx.sspl.sspl_test.run_qa_test import RunQATest
             # RunQATest(self.plan).run()
             try:
-                CMD = "%s/run_qa_test.sh --plan %s" % (TEST_DIR, self.plan)
+                CMD = "%s/run_qa_test.sh --plan %s" % (self.test_dir, self.plan)
+                if self.args.v2:
+                    CMD = "%s/run_tests.sh --plan %s" % (self.test_dir, self.plan)
```

Comment:
CMD above if clause and in if clause is same. So, why if clause is required ?

Reply:
run_qa_test.sh vs run_tests.sh